wcivf/apps/elections/import_helpers.py

```diff
@@ -299,7 +305,9 @@ def do_import(self, params=None):
         if self.should_run_post_ballot_import_tasks:
             self.attach_cancelled_ballot_info()
 
-        if self.recently_updated:
+        qs = self.ee_helper.iter_recently_modified_election_ids(group_type="election")
+
+        if self.recently_updated or qs:
```

Comment:
I'm unsure this is the way to go as I think this will always return as True. I want to build a qs that checks for recently modified elections in EE.